packages/single-spa-welcome/src/welcome.js

```diff
@@ -1,7 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import "./welcome.css";
 
 export default function Root(props) {
+  const isUsingSingleSpaLayout = useState(() => {
+    // Search the dom for a single-spa-router element
+    // While this won't work for SSR, the goal of this app is to help
+    // beginners, which likely won't use SSR before running their
+    // first root-config.
+    const templates = Array.prototype.slice.call(
+      document.querySelectorAll("template")
+    );
+
+    return templates.some((template) => {
+      const hasSingleSpaRouter = !!template.content.querySelector(
+        "single-spa-router"
+      );
+      return hasSingleSpaRouter;
+    });
+  })[0];
```

Comment:
I did this because jest was complaining about Babel's array destructuring helper that was esm and I didn't want to go through the effort of figuring that out.